src/sqllogictest/src/runner.rs

```diff
@@ -1175,17 +1175,6 @@ impl RunnerInner {
             [statement] => statement,
             _ => bail!("Got multiple statements: {:?}", statements),
         };
-        match statement {
-            Statement::CreateView { .. }
-            | Statement::Select { .. }
-            | Statement::Show(ShowStatement::ShowObjects(..)) => (),
-            _ => {
-                if output.is_err() {
-                    // We're not interested in testing our hacky handling of INSERT etc
```

Comment:
@benesch any idea what this comment means? My only guess is that it's from a time when `INSERT` wasn't implemented?

Reply:
That's probably from 3 years ago when INSERT was very different. Is almost certainly wildly outdated and misleading.

Reply:
Yep, Matt's right.

Reply:
Yes, previously INSERT ... SELECT was not implemented in Mz. Instead, it was sent off to Postgres and the result re-ingested from there.